app/src/main/java/com/example/habitsmasher/ui/history/HabitEventViewFragment.java

```diff
@@ -44,11 +62,38 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         // Grab text boxes
         TextView eventCommentTextBox = view.findViewById(R.id.view_event_comment);
         TextView eventDateTextBox = view.findViewById(R.id.view_event_date);
+        _eventImageView = view.findViewById(R.id.view_event_image);
 
         // Setting text boxes
         eventCommentTextBox.setText(_habitEvent.getComment());
         eventDateTextBox.setText(simpleDateFormat.format(_habitEvent.getDate()));
 
+        // Set image
+        fetchEventImageFromDB();
+
         return view;
     }
+
+    private void fetchEventImageFromDB() {
+        FirebaseStorage storage = FirebaseStorage.getInstance();
+        StorageReference storageReference = storage.getReference();
+
+        StorageReference ref = storageReference.child("images/" + _userId + "/" + _parentHabit.getId() + "/" + _habitEvent.getId() + "/eventImage");
```

Comment:
Any way to fetch the path from a helper method instead of building it up every time?